Mathlib/NumberTheory/LSeries/Linearity.lean

```diff
@@ -132,3 +130,37 @@ lemma LSeriesSummable.smul_iff {f : ℕ → ℂ} {c s : ℂ} (hc : c ≠ 0) :
 @[simp]
 lemma LSeries_smul (f : ℕ → ℂ) (c s : ℂ) : LSeries (c • f) s = c * LSeries f s := by
   simp only [LSeries, term_smul_apply, tsum_mul_left]
+
+/-!
+### Sums
+-/
+
+section sum
+
+variable {ι : Type*} (f : ι → ℕ → ℂ) (S : Finset ι) (s : ℂ)
+
+lemma LSeries.term_sum_apply (n : ℕ) :
```

Comment:
`@[simp]`?

Reply:
It doesn't hurt, and follows the precedent of `Finset.sum_apply`.